pom.xml

```diff
@@ -575,6 +562,7 @@ under the License.
           <configuration>
             <excludes>
               <exclude>.asf.yaml</exclude>
+              <exclude>.mvn/</exclude>
```

Comment:
Nit: this is probably unrelated

Reply:
Um, no, as part of No2 experiment w/ provided checksums, this PR contains `.mvn/checksums` that makes RAT freak out :smile:

Reply:
I see.
Makes sense